src/trait.js

```diff
@@ -9,6 +9,7 @@ var parser = require('php-parser');
 var filter = require('./helpers/filter');
 var Method = require('./method');
 var Constant = require('./constant');
```

Comment:
could be better to remove also the require

Reply:
Yes, sorry, I must have not noticed this.

Reply:
It does not matters, you've fixed the most important part here :smile:, your commit was just perfect. Also, thanks for tests, I've passed from 50% to 70% :+1: